llvm/test/Transforms/InstSimplify/ConstProp/bitcast.ll

```diff
@@ -66,15 +66,17 @@ define i1 @fcmp_constexpr_une(float %conv) {
 
 define i1 @fcmp_constexpr_ueq(float %conv) {
 ; CHECK-LABEL: @fcmp_constexpr_ueq(
-; CHECK-NEXT:    ret i1 true
+; CHECK-NEXT:    [[CMP:%.*]] = fcmp ueq float bitcast (i32 ptrtoint (ptr @a to i32) to float), bitcast (i32 ptrtoint (ptr @a to i32) to float)
```

Comment:
Shouldn't we still be able to fold this? I think for the purpose of denormal flushing, you probably want to ignore constant expressions rather than bail out for them, so they can still be evaluated symbolically.